api/v1alpha1/podtypes/access_policy.go

```diff
@@ -27,6 +31,8 @@ type InternalRule struct {
 	Namespace string `json:"namespace,omitempty"`
 	//+kubebuilder:validation:Required
 	Application string `json:"application"`
+	//+kubebuilder:validation:Optional
+	NamespacesByLabel *metav1.LabelSelector `json:"namespaceByLabel,omitempty"`
```

Comment:
mismatch in the struct definition name and the json key

Reply:
Thanks, fixed